src/api.js

```diff
@@ -0,0 +1,33 @@
+const getTrendingAlbums = (setRecommendedData) => {
+    return fetch('https://api.discogs.com/database/search?type=master&format=vinyl&key=GimREdkHlKcSjALMSwEP&secret=RZbpExNDRyTdbTAaiVxiJpiYgOcydrMJ&year=2023&country=US&page=1&per_page=5&sort=hot', {mode:'cors'})
+        .then(res => {
+        if (res.ok) {
+            return res.json()
+        }
+        throw new Error('Something went wrong')
+        })
+        .then(data => setRecommendedData(data))
+        .catch(err => console.error(err))
+}
+
+const getRecommendedAlbums = (albumID, setRecommendedData) => {
+    return fetch(`https://api.discogs.com/masters/${albumID}?key=GimREdkHlKcSjALMSwEP&secret=RZbpExNDRyTdbTAaiVxiJpiYgOcydrMJ`)
+        .then(res => {
+          if (res.ok) {
+            return res.json()
+          }
+          throw new Error('Something went wrong')
+        })
+        .then(data => fetch(`https://api.discogs.com/database/search?type=master&format=vinyl&key=GimREdkHlKcSjALMSwEP&secret=RZbpExNDRyTdbTAaiVxiJpiYgOcydrMJ&year=2023&country=US&page=1&per_page=5&genre=${data.genres[0]}`, {mode:'cors'}))
```

Comment:
I put this in because i was getting a phantom console error saying my request was block and it seemed to fix it.  I just removed it and there wasn't an error, so not sure what happened 😐